packages/go_router_builder/lib/src/path_utils.dart

```diff
@@ -87,74 +85,3 @@ String patternToPath(String pattern, Map<String, String> pathParameters) {
   }
   return buffer.toString();
 }
-
-/// Extracts arguments from the `match` and maps them by parameter name.
-///
-/// The [parameters] should originate from the call to [patternToRegExp] that
-/// creates the [RegExp].
-Map<String, String> extractPathParameters(
-    List<String> parameters, RegExpMatch match) {
-  return <String, String>{
-    for (int i = 0; i < parameters.length; ++i)
-      parameters[i]: match.namedGroup(parameters[i])!
-  };
-}
-
-/// Concatenates two paths.
-///
```

Comment:
In this commit ([revamp: Remove unused methods](https://github.com/flutter/packages/pull/4524/commits/0cc50db8b064e6b26629891a16710ae3272af622)) I'm only deleting methods that are not used